<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>641</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter Normes</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>481</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Identifiant</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>119</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_description">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>80</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="le_date">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>DateCreation</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher Normes</string>
     </attribute>
     <widget class="QTableView" name="tab_normes">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>531</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxSort">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Supprimer Normes</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_supp">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Rechercher</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>681</width>
        <height>321</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Recherche Par Id</string>
       </attribute>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_id_recherche">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_rechercher_id">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>30</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QTableView" name="tab_recherche_id">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>461</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Recherche Par Nom</string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nom_recherche">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_rechercher_nom">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>40</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QTableView" name="tab_recherche_nom">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>491</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Exporter Pdf</string>
     </attribute>
     <widget class="QPushButton" name="pb_exporter">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Exporter</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
